server/modules/auth/auth.middleware.ts

```
@@ -0,0 +1,135 @@
+) => {
+  let token;
+
+  // Get token from cookies
+  if (req.cookies.token) {
+    token = req.cookies.token;
+  }
+
+  // Check Authorization header as fallback
+  else if (
+    req.headers.authorization &&
+    req.headers.authorization.startsWith('Bearer')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)